src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+  },
```

Comment:
- [ ] Great job on creating the bookstore! :+1: However, as a project requirement, it is necessary to import the reducer for categories.